libr/core/cconfig.c

```diff
@@ -3998,7 +3998,7 @@ R_API int r_core_config_init(RCore *core) {
 		if (!found) {
 			SETCB ("prj.vc.type", "git", &cb_prjvctype, "What should projects use as a vc");
 		} else {
-			SETCB ("prj.vc.type", "git", &cb_prjvctype, "What should projects use as a vc");
+			SETCB ("prj.vc.type", "rvc", &cb_prjvctype, "What should projects use as a vc");
```

Comment:
I dont think we want to use rvc as default vc yet. Bring back git and remove the warning message